requirements.txt

```diff
@@ -16,6 +16,7 @@ django-countries==7.5
 django-extensions==3.2.1
 django-file-form==3.4.3
 django-filter==2.4.0
+django-formtools==2.4.1
```

Comment:
Looks like you adding `django-formtools` as new dependency and not really upgrading. 

Do you think it was removed somehow?